sources/updater/updog/src/main.rs

```diff
@@ -42,13 +44,16 @@ enum Command {
     Update,
     UpdateImage,
     UpdateApply,
+    RevertUpdateApply,
 }
 
 #[derive(Debug, Deserialize)]
 struct Config {
     metadata_base_url: String,
     targets_base_url: String,
     seed: u32,
+    version_lock: String,
+    ignore_waves: bool,
     // TODO API sourced configuration, eg.
     // blacklist: Option<Vec<Version>>,
     // mode: Option<{Automatic, Managed, Disabled}>
```

Comment:
Have we changed this before? The file gets changed based on api settings such that it can never be of the wrong shape at boot?

```rust
#[derive(Debug, Deserialize)]
struct Config {
    metadata_base_url: String,
    targets_base_url: String,
    seed: u32,
    version_lock: String,
    ignore_waves: bool,
    // TODO API sourced configuration, eg.
    // blacklist: Option<Vec<Version>>,
    // mode: Option<{Automatic, Managed, Disabled}>
```

Reply:
The updog config has changed before when we added `seed`. This config has to mirror the `thar-be-settings` generated `updog.toml`
`blacklist` and `mode` don't exist in the API settings yet if that's what you're asking.

Reply:
Yeah thanks that helps.